src/lib/hooks/useConnect/__test__/setupConnection.spec.ts

```diff
@@ -2,32 +2,27 @@
 /* eslint-disable global-require */
 import { SDK_ACTIONS } from '../../../dux/sdk/actionTypes';
 import { USER_ACTIONS } from '../../../dux/user/actionTypes';
-import { getMissingParamError, setUpConnection, setUpParams } from '../setupConnection';
+import { getMissingParamError, setUpConnection, initSDK, getConnectSbError } from '../setupConnection';
 import { SetupConnectionTypes } from '../types';
 import { generateSetUpConnectionParams, mockSdk, mockUser, mockUser2 } from './data.mocks';
-
 import { SendbirdError } from '@sendbird/chat';
 
 // todo: combine after mock-sdk is implemented
-jest.mock('@sendbird/chat', () => ({
-  init: jest.fn().mockImplementation(() => mockSdk),
-  SendbirdError: jest.fn(),
-}));
-jest.mock('@sendbird/chat/openChannel', () => ({
-  OpenChannelModule: jest.fn(),
-}));
-jest.mock('@sendbird/chat/groupChannel', () => ({
-  GroupChannelModule: jest.fn(),
-}));
+jest.mock('@sendbird/chat', () => {
+  const originalModule = jest.requireActual('@sendbird/chat');
```

Comment:
why did we have to change to requireActual?

Reply:
If the rest of the module is not re-added, only the mocked method will be returned. (If there is code that uses the rest of the module, an undefined error will occur.)

Reply:
If such an error occurs, the test code needs to be modified.
Previously, the actual code included logic to avoid this error(`SendbirdProduct?.UIKIT_CHAT ?? 'uikit-chat'`)
which made the code somewhat complex.